integration/update_array_compat_test.go

```diff
@@ -238,3 +238,61 @@ func TestUpdateArrayCompatPullAll(t *testing.T) {
 
 	testUpdateCompat(t, testCases)
 }
+
+func TestUpdateArrayCompatAddToSetEach(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]updateCompatTestCase{
+		"Document": {
+			update: bson.D{{"$addToSet", bson.D{{"v", bson.D{
+				{"$each", bson.A{bson.D{{"field", int32(42)}}}},
+			}}}}},
+			skipForTigris: "Mixed types are not supported for Tigris.",
+		},
+		"String": {
+			update: bson.D{{"$addToSet", bson.D{{"v", bson.D{{"$each", bson.A{"foo"}}}}}}},
+		},
+		"Int32": {
+			update: bson.D{{"$addToSet", bson.D{{"v", bson.D{
+				{"$each", bson.A{int32(1), int32(42), int32(2)}},
+			}}}}},
+			skipForTigris: "Mixed types are not supported for Tigris.",
+		},
+		"NotArray": {
+			update:     bson.D{{"$addToSet", bson.D{{"v", bson.D{{"$each", int32(1)}}}}}},
+			resultType: emptyResult,
+		},
+		"EmptyArray": {
+			filter:     bson.D{{"_id", "array-documents-nested"}},
+			update:     bson.D{{"$addToSet", bson.D{{"v", bson.D{{"$each", bson.A{}}}}}}},
+			resultType: emptyResult,
+		},
+		"ArrayValuesExists": {
+			update:        bson.D{{"$addToSet", bson.D{{"v", bson.D{{"$each", bson.A{int32(42), int32(43)}}}}}}},
+			skipForTigris: "Tigris schema validation would fail.",
```

Comment:
`skipForTigris` reason is not correct, the test case is quite similar to `Int32` test case, what value does it bring?

Reply:
Fixed.